app/models/search_timdex.rb

```diff
@@ -0,0 +1,35 @@
+# Searches TIMDEX! API and formats results into {Result} object
+#
+# == Required Environment Variables:
+# - TIMDEX_URL
+#
+# == Optional Environment Variables:
+# - RESULTS_PER_BOX
+#
+# == See Also
+# - https://timdex.mit.edu/playground
+class SearchTimdex
+  attr_reader :results
+
+  TIMDEX_URL = ENV['TIMDEX_URL'].freeze
+
+  def initialize
+    @timdex_http = HTTP.persistent(TIMDEX_URL)
+    @results = {}
+  end
+
+  # Run a search
+  # @param term [string] The string we are searching for
+  # @return [Hash] A Hash with search metadata and an Array of {Result}s
+  def search(term)
+    @query = '{"query":"{search(searchterm: \"' + term + '\", source: \"MIT ArchivesSpace\") {hits records {sourceLink title summary} } }"}'
+    results = HTTP.headers(accept: 'application/json',
+        'Accept-Encoding': 'gzip, deflate, br',
```

Comment:
As we are not going to switch to the GraphQL gem I want to revisit the flags here. This is not a comment to change anything, it's mostly a note to myself to actually check these settings which I mostly skimmed thinking we might rip this all out before it landed anyway which is no longer the plan :)